core/src/main/java/com/linecorp/armeria/server/annotation/FallthroughException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.annotation;
+
+import com.linecorp.armeria.common.Flags;
+import com.linecorp.armeria.common.util.Exceptions;
+
+/**
+ * A {@link RuntimeException} raised for falling through to the next something. It would be raised from
+ * {@link ExceptionHandlerFunction}, {@link RequestConverterFunction} and/or {@link ResponseConverterFunction}.
+ */
+public final class FallthroughException extends RuntimeException {
```

Comment:
Probably OK to hide this from the public API?

Reply:
Ok! Changed to package-local. Also, removed public `get()` method and renamed `singleton()` to `get()`.